### PR TITLE
Changes to quickstart notebook

### PR DESCRIPTION
**Pull request by kaimartin365**
I changed the xlim and y-axis name.
**Description**

The xlim was changed from 3000 to 9000 to 500 to 9000. The y-axis name was changed from Luminosity to Luminosity Density.
**Motivation and context**

**How has this been tested?**
- [ ] Testing pipeline.
- [x] Other. 

**Examples**

**Type of change**

- [ ] Bug fix. 
- [ ] New feature. 
- [ ] Breaking change. 
- [x] None of the above. 

**Checklist**

- [ ] My change requires a change to the documentation.
    - [ ] I have updated the documentation accordingly.
    - [ ] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [x] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

File: docs/quickstart/quickstart.ipynb [hunk 1]
```diff
@@ -133,7 +133,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.3"
+   "version": "3.7.10"
```
Comment (KevinCawley) [inline docs/quickstart/quickstart.ipynb:136]: For this issue we only want to change the ylabel and xlim, so you will want to undo this change.

Comment (smithis7) [reply docs/quickstart/quickstart.ipynb:136]: That's just metadata, the only way he can change that is to update his python

Comment (KevinCawley) [reply docs/quickstart/quickstart.ipynb:136]: Can this be editted in the file with a commit so that it will be reverted? I was under the impression that only the graph edits.

Comment (epassaro) [reply docs/quickstart/quickstart.ipynb:136]: It's ok. @kaimartin365 used the newest TARDIS environment which reverted Python to 3.7. The metadata always reflect the last Python version used to render that notebook, even when you clear all the notebook cells.

**Review by KevinCawley — APPROVED** [review]

**Review by smithis7 — APPROVED** [review]

**Comment by smithis7** [comment]
I have approved -- it can't be merged though until you mark it ready for review.

**Review by Rodot- — APPROVED** [review]

**Comment by andrewfullard** [comment]
This needs to be rebased
